lib/debugger-wrapper.js

```diff
@@ -71,22 +71,21 @@ function instrumentScript(scriptUrl) {
 function setBreakpointOnFunction(scriptUrl, moduleInfo, functionName, functionLocation) {
   const breakpointParameters = {
     lineNumber: functionLocation.start.line - 1, // lines are 0-based in the inspector but 1-based in acorn
-    columnNumber: functionLocation.start.column, // columns are 0-based in both places (?)
+    columnNumber: functionLocation.start.column - 1, // same for columns
```

Comment:
we have many many tests testing the logic here. we never minded this 1 character offset because the debugger always compensated for our mistake, and it still does - actually even this isn't 100% correct, because we probably would want to find the "correct" place for a breakpoint ourselves - but this is a discussion we already had when we talked about comparing the "requested" and "actual" bp location, and realised it's not a trivial task.
so I don't really think there's something additional we can do here for now. anything you can see?